test/Test/Db.hs

```diff
@@ -1,2 +1,79 @@
-module Test.Db where
-    
\ No newline at end of file
+module Test.Db 
+       ( dbSpecs
+       ) where
+
+import IW.App (AppEnv)
+import IW.Core.Id (Id (..))
+import IW.Core.Issue (Issue (..))
+import IW.Core.Repo (RepoName (..), RepoOwner (..))
+import IW.Core.SqlArray (SqlArray (..))
+import IW.Db (getIssues, prepareDb, upsertIssues)
+import IW.Effects.Log (runAppLogIO_)
+
+import Test.Assert (equals, equals_)
+import Test.Common
+import Test.Hspec (Spec, before_, describe, it)
+
+
+dbSpecs :: AppEnv -> Spec
+dbSpecs = joinSpecs "DB"
+    [ getIssuesSpec
+    , upsertIssuesSpec
+    ]
+
+getIssuesSpec :: AppEnv -> Spec
+getIssuesSpec env = describe "getIssues function" $
+    it "should return a list of length 1" $
+        env & (length <$> getIssues) `equals` 1
+
+upsertIssuesSpec :: AppEnv -> Spec
+upsertIssuesSpec env = before_ (runAppLogIO_ env prepareDb) $
```

Comment:
I've been using `before_` to prepare the database before each test. So I get a fresh database each time. Is this not needed?

Reply:
I see. This command makes tests slower. So we only need to call it when necessary, not before each test. So subsequent tests will work over the result of previous tests. For example, once you add tests for `Repo`, it should be possible to make `seed.sql` file completely empty because all required data will be inserted via tests.